packages/office-ui-fabric-react/scripts/loadTheme.js

```diff
@@ -0,0 +1,43 @@
+const { getTheme } = require('@uifabric/styling');
```

Comment:
I think you'll probably want to have a set of verify & update functions for these files. So prepare yourself to have to split the code here in this script in another C/I perhaps.